scripts/serverless.js

```diff
@@ -474,6 +474,8 @@ processSpanPromise = (async () => {
           if (!ensureResolvedProperty('frameworkVersion')) return;
           if (!ensureResolvedProperty('app')) return;
           if (!ensureResolvedProperty('org')) return;
+          if (!ensureResolvedProperty('dashboard')) return;
```

Comment:
I figured it would be good to ensure resolution of these properties as we rely on them moving forward e.g. in `isDashboardEnabled` check we rely on `dashboard` property. Please let me know what do you think @medikoo

Reply:
Yes, it's great you added those here. I think it's fine to require `dashboard` and `console` configuration to be fully resolved at this point